docs/install-alternatives.md

```diff
@@ -15,28 +15,66 @@ notes below.
 
 <div class="p-notification--information">
   <p class="p-notification__response">
-    Installing MicroK8s with Multipass requires Windows 10 Professional or
-    Windows 10 Enterprise, as it relies on the <em>Hyper-V</em> hypervisor. It
+    Installing MicroK8s with Multipass requires Windows 10 Professional,
+    Windows 10 Education or Windows 10 Enterprise, as it relies on the
+    <em>Hyper-V</em> hypervisor. It
     will also require at least 4GB of available RAM and 40GB of storage.
   </p>
 </div>
 
-Although Windows 10 now has some very useful features, such as the ability to
-install [Ubuntu as an app][ubuntu-app], the integration of WSL2 still
-doesn't provide all the Ubuntu functionality required to make MicroK8s run
-smoothly out-of-the-box.
+From 1.18, MicroK8s now has an official Windows installer, which is the
+recommended way to install MicroK8s
 
-If you wish to experiment with running MicroK8s semi-natively, take a look at
-this [discourse post on WSL2][windows-post].
+1. **Enable Hyper-V**
+    If you have not previously done so, you should enable the Hyper-V
+    hypervisor feature in Windows 10. To do so, please follow the
+    instructions on the [MicroSoft website][hyper-v].
 
-For now, the best way to run MicroK8s on Windows is with virtualisation.
-MicroK8s will install without problems on Ubuntu running on number of different
-VMs, including [VirtualBox](https://www.virtualbox.org/).
+1.  **Download the installer**
+
+    The Windows installer is available on the MicroK8s GitHub page.
+    [Download it here](https://github.com/ubuntu/microk8s/releases/download/installer-v1.0.0/microk8s-installer.exe)
```

Comment:
This is a pre-release, do you want to hold off until the release officially happens?

Reply:
thanks for the heads up, I'll sit on it for now

Reply:
I believe this has been released now?